src/pr.ts

```diff
@@ -29,8 +30,11 @@ export const raisePR = async (forkBranchName: string, targetBranch: string, extr
     title: `chore: bump libcc (${targetBranch})`,
     body: `Updating libcc reference to latest.  Changes since the last roll:
 
-${extraCommits.map((commit) =>
-      `* [\`${commit.sha.substr(0, 8)}\`](${COMMIT_URL_BASE}/${commit.sha}) ${commit.message}`).join('\n')}
+${extraCommits.map((commit) => {
+      const sha = `[\`${commit.sha.substr(0, 8)}\`](${COMMIT_URL_BASE}${commit.sha})`;
+      const msg = commit.message.replace(/#(\d+)/g, `${ISSUE_URL_BASE}$1`);
+      return `* ${sha} ${msg}`;
```

Comment:
Not all commits will reference a PR / issue, we should check that the commit message actually matches before doing this insertion

Reply:
I belive this PR doesn't change the original behavior when the regex doesn't match. Should I add if statement with `Regex`'s `test` method?